src/taipy/templates/multi-page-gui/{{cookiecutter.application_name}}/requirements.txt

```diff
@@ -0,0 +1 @@
+taipy>=2.3
```

Comment:
Oh!
A taipy version depends on a taipy-templates version that declares a taipy version.
This is bad...

The taipy version should be generated on the fly depending on the taipy package used to scaffold an application.

I don't know how easy it is to fix it.

Reply:
I think it is doable with a hook. I will propose something